doc/cookbook/resilience.md

```diff
@@ -3,12 +3,12 @@
 - [Resilience](#resilience)
   - [Basic: Load Balance](#basic-load-balance)
   - [More Livingness: Resilience of Service](#more-livingness-resilience-of-service)
-    - [Circuit Break](#circuit-break)
+    - [CircuitBreaker](#circuitbreaker)
     - [RateLimiter](#ratelimiter)
     - [Retry](#retry)
     - [Time Limit](#time-limit)
```

Comment:
How about change `Time Limit` to `TimeLimiter` by the way

Reply:
There is no `TimeLimiter` in easegress 2.0. We use a simple timeout parameter in proxy.

Reply:
updated!